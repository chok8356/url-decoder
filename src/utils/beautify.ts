import jsBeautify from 'js-beautify';

export const beautify = (str: string) => {
  return jsBeautify(str, { indent_size: 2 });
};
