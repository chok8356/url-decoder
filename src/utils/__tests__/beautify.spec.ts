import { beautify } from '../beautify';

const data = '{a: 1, b:{c: true}}';

describe('Utils: beautify', () => {
  it('render', () => {
    expect(beautify(data)).toMatchSnapshot();
  });
});
