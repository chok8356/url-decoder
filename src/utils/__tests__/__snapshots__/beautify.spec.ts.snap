// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Utils: beautify render 1`] = `
"{
  a: 1,
  b: {
    c: true
  }
}"
`;
